pilot/pkg/serviceregistry/mcp/controller.go

```diff
@@ -125,6 +133,53 @@ func (c *controller) List(typ resource.GroupVersionKind, namespace string) (out
 	return out, nil
 }
 
+func (c *controller) Distribute(group string, s *snapshotter.Snapshot) {
+	switch group {
+	case snapshots.Istiod, snapshots.SyntheticServiceEntry:
```

Comment:
Why do this for SSEs?

Reply:
The reason SSE is ignored because we don't want to handle any SSE in this MCP controller since there is a dedicated SSE controller.